### ISSUE TITLE
babel ignores transpiling of const

### ISSUE THREAD

**Issue by quebits** [dosentmatter/babel-plugin-const-enum#18]
When this plugin configured with `"transform": "constObject"` and babel use default "preset-env" + "preset-typescript" it cause that result of transpilation stay intact.

For example:

```typescript
const enum SomeEnum {
  Entry1 = 'key'
}

const someVar = 'test'
```

after transpilation turns into:

```typescript
const SomeEnum = {
  Entry1: "key"
}

var someVar = 'test'
```

Ex enum is not transpiling down to `var SomeEnum = { ... }`.
Look like that this code prevents further transpilation:
https://github.com/dosentmatter/babel-plugin-const-enum/blob/77fdb780a34d49b1061463b1de30c1ef64f04124/src/const-object.js#L17

**Comment by dosentmatter** [comment]
@krail, sorry for the delay. The fix is merged in master now. I will publish to npm soon.

**Comment by dosentmatter** [comment]
The fix was published on version `1.0.0`
